sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslateContent.Serialization.cs

```diff
@@ -27,14 +27,15 @@ internal virtual MultipartFormDataRequestContent ToMultipartRequestContent()
             return content;
         }
 
-        void global::System.ClientModel.Primitives.IJsonModel<global::Azure.AI.Translation.Document.DocumentTranslateContent>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
+        /// <param name="writer"> The JSON writer. </param>
+        /// <param name="options"> The client options for reading and writing models. </param>
+        protected virtual void JsonModelWriteCore(Utf8JsonWriter writer, ModelReaderWriterOptions options)
         {
             var format = options.Format == "W" ? ((IPersistableModel<DocumentTranslateContent>)this).GetFormatFromOptions(options) : options.Format;
             if (format != "J")
             {
                 throw new FormatException($"The model {nameof(DocumentTranslateContent)} does not support writing '{format}' format.");
             }
-
             writer.WriteStartObject();
             writer.WritePropertyName("document"u8);
```

Comment:
this change is not correct.
Our design is:
The implementation of the interface (Write method) will call the WriteCore method, wrapping it by `WriteStartObject` and `WriteEndObject`, and the implementation of the WriteCore method should NOT contain the `WriteStartObject` and `WriteEndObject`.
Therefore this part needs to change to remove the `WriteStartObject` and `WriteEndObject`.

Reply:
Fixed